style : update back to top button design and functionality

lirena00 opened the pull request:
## Rationale

This PR is aimed to fix the things mentioned in #112. It was mentioned that the layout seem odd when back to top and pagination are aligned centered
I tried to fix this by moving the backtotop button to the side with a more modern feel to the button.


![image](https://github.com/user-attachments/assets/33c13976-d558-4817-99ac-ce08eb05d712)


fix #112 

## Changes
- backtotop is now a circular button and is fixed at side
- used icons instead of arrow characters that were being used previously for pagination and backtotop button

benoit74 reviewed (changes requested):
Thanks, mostly OK.

Please:
- add an entry in CHANGELOG.md (see existing entries for expected format)
- fix your first comment so that issue and PR are properly linked (https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue)

lirena00 commented:
@benoit74 I did used `fix #<issue_no>` in the comment though or you want me to use that at the top of comment ? also I made an entry in the changelogs like you told.

benoit74 commented:
Sorry, I did not saw that below the hugs image ^^ 
Changelog must preferably mention issue number, not PR number.
Please fix that and squash all your commits into one.
Thank you

lirena00 commented:
@benoit74  actually I am having a problem while squashing I am not entirely sure how to deal with it so please help if you can 
thing is when I first tried to squash the commits it also showed a commit made for `--link` flag and if i squashed that too in between it would result in merging of those changes in this commit
then I tried to hard reset my `newstyle` branch of my form with `main` and then redid the changes and commited them, still there were 3 commits that were shown on the `commits` section  which also included the minimum height commit I made a few days ago even though I have squashed all of them but still I feel I am getting lost, was there a better way? I tried to find it online but its getting even more confusing

lirena00 commented:
ok no problem figured the thing out, definitely learned something new today bout git 😅 everything is now as it should be

benoit74 reviewed (approved):
LGTM
